syft/pkg/type.go

```diff
@@ -14,6 +14,7 @@ const (
 	RpmPkg           Type = "rpm"
 	NpmPkg           Type = "npm"
 	PythonPkg        Type = "python"
+	PhpPkg           Type = "php"
```

Comment:
we try to keep package names as close to the package manager in question (when possible). 

```suggestion
	PhpComposerPkg           Type = "php-composer"
```